pkg/cmd/kafka/create/create.go

```diff
@@ -582,7 +612,7 @@ func promptKafkaPayload(opts *options, constants *remote.DynamicServiceConstants
 	}
 
 	// If the user is not using an enterprise cluster, prompt them to select a cloud provider
-	if !opts.useEnterpriseCluster {
+	if opts.hasLegacyQuota && opts.useLegacyFlow {
```

Comment:
Nit pick but couldn't this be just `opts.useLegacyFlow` as if that is true then `opts.hasLegacyQuota` is always true

Reply:
makes sense 👍